pkg/relic/Makefile

```diff
@@ -9,17 +9,13 @@ all: $(PKG_BUILDDIR)/Makefile
 	"$(MAKE)" -C $(PKG_BUILDDIR) && \
 	cp $(PKG_BUILDDIR)/lib/librelic_s.a $(BINDIR)/$(PKG_NAME).a
 
-$(PKG_BUILDDIR)/comp-options.cmake: fix_source
+$(PKG_BUILDDIR)/comp-options.cmake: git-download
 	cd "$(PKG_BUILDDIR)" && perl $(PKG_DIR)/generate-cmake-xcompile.perl > comp-options.cmake
 
 $(PKG_BUILDDIR)/Makefile: $(PKG_BUILDDIR)/comp-options.cmake
 	cd "$(PKG_BUILDDIR)" && COMP="$(filter-out -Werror=old-style-definition -Werror=strict-prototypes, $(CFLAGS) ) " cmake -DCMAKE_TOOLCHAIN_FILE=comp-options.cmake -DCHECK=off -DTESTS=0 -DBENCH=0 -DSHLIB=off -Wno-dev $(RELIC_CONFIG_FLAGS) .
 
-CFLAGS += -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-function
```

Comment:
@tfar any progress? I just tested and the only problem is with `variadic-macros`, so if you (re)add the corresponding CFLAGS (as above) to disable the errors it works fine.

If you don't find the time right now to work on this, I'm willing to overtake - if you don't mind?